Makefile

```diff
@@ -153,8 +153,8 @@ test_format: format
 	@! git diff --name-only --diff-filter=ACMRT | xargs -n1 clang-format -style=file -output-replacements-xml | grep '<replacements' > /dev/null; if [ $$? -ne 0 ] ; then git --no-pager diff && exit 1 ; fi
 
 # Builds the project or library
-ifneq (,$(PROJECT))
-build: $(BIN_DIR)/$(PROJECT)$(PLATFORM_EXT)
+ifneq (,$(PROJECT)$(TEST))
+build: $(GDB_TARGET)
```

Comment:
Nit: This is a bad name for this variable if it is used for non GDB stuff.

Reply:
`TARGET_EXECUTABLE/APPLICATION/BINARY`?

Reply:
`TARGET_BINARY` is probably good.